transform/mattermost-analytics/models/intermediate/product/nps/int_nps_score.sql

```diff
@@ -0,0 +1,49 @@
+with mattermost_nps as (
+select 
+    server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , score as score
+    , user_role as user_role
+    , received_at as score_received_at
+    from {{ ref('stg_mattermost_nps__nps_score') }} 
+    qualify ROW_NUMBER() over (PARTITION BY event_date, user_id ORDER BY timestamp DESC) = 1
```

Comment:
nit: using lower-case to be aligned with the rest of the code.

Reply:
`server_id` should be added as part of the partitioning for clarity.

Reply:
Added `server_id` to partition